src/main/java/core/basesyntax/strategy/operation/OperationHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.strategy.operation;
+
+public interface OperationHandler {
+    void operationType(String fruit, int amount);
```

Comment:
The method name 'operationType' is not descriptive enough. Consider renaming it to something more indicative of the action it performs, like 'applyOperation' or 'handleOperation'.